### ISSUE TITLE
Bug when trying to use "flatten"?

### ISSUE THREAD

**Issue by dasnessie** [tstehr/RecipeMD#36]
I got an unhandled exception:

```
$ recipemd -f chili-kaese-spaetzle.md 
Traceback (most recent call last):
  File "/usr/bin/recipemd", line 8, in <module>
    sys.exit(main())
  File "/home/nessie/.local/lib/python3.9/site-packages/recipemd/cli/main.py", line 55, in main
    r = _get_flattened_recipe(r, recipe_url=recipe_url, parser=rp)
  File "/home/nessie/.local/lib/python3.9/site-packages/recipemd/cli/main.py", line 151, in _get_flattened_recipe
    recipe = _create_flattened_substituted_ingredients(recipe, ingr_to_recipe)
  File "/home/nessie/.local/lib/python3.9/site-packages/recipemd/cli/main.py", line 246, in _create_flattened_substituted_ingredients
    flattened_ingr = _create_flattened_substituted_ingredients(ingr, ingr_to_recipe)
  File "/home/nessie/.local/lib/python3.9/site-packages/recipemd/cli/main.py", line 254, in _create_flattened_substituted_ingredients
    return replace(
TypeError: replace() missing 1 required positional argument: 'obj'
```

**Comment by tstehr** [comment]
Thank you for reporting 👍

This error is due to a change to [replace](https://docs.python.org/3/library/dataclasses.html#dataclasses.replace) in Python 3.9. A [fix](https://github.com/tstehr/RecipeMD/commit/186618df1f9e6b396ffc4b3a7b00cd2f8cdb6639) is already on master and will be part of the next release.

**Comment by tstehr** [comment]
[Version 4.0.7](https://pypi.org/project/recipemd/4.0.7/) is now live and includes the fix for `--flatten` on Python 3.9.
